jupyterhub/apihandlers/users.py

```diff
@@ -587,6 +588,139 @@ def on_finish(self):
             await self.send_event(failed_event)
 
 
+def _parse_timestamp(timestamp):
+    """Parse and return a utc timestamp
+
+    - raise HTTPError(400) on parse error
+    - handle and strip tz info for internal consistency
+      (we use naïve utc timestamps everywhere)
```

Comment:
Not sure if the special character will render properly in autodoc but it may.

Reply:
it should, since we're running sphinx with Python 3